pkg/flowkit/config/processor.go

```diff
@@ -43,8 +43,14 @@ func ProcessorRun(raw []byte) ([]byte, map[string]string) {
 
 // processEnv finds env variables and insert env values.
 func processEnv(raw string) string {
-	_ = godotenv.Load() // try to load .env file
-
+	path, _ := os.Getwd()
+	envFilePath := path + "/private_keys.env"
+	_, error := os.Stat(envFilePath)
+	// check if error is not "file not exists"
+	if error == nil {
+		godotenv.Load(envFilePath)
+	}
```

Comment:
Will always load private_keys.env and read the env variables whenever we start up the command